sklearn/tree/benchmark_iforest.py

```diff
@@ -0,0 +1,67 @@
+"""
+==========================================
+IForest benchmark
+==========================================
+
+A test of IForest on classical anomaly detection datasets.
+
+"""
+print(__doc__)
+
+import numpy as np
+import pandas as pd
+from iforest import *
+from sklearn.metrics import roc_curve
+import matplotlib.pyplot as plt
+from sklearn import svm
+from sklearn import grid_search
+
+
+### KDD99 datasets: http-le, http10-le (=10% http-le), smtp-le, SF-le, SA-le
+# or with '-lb' or'3d' instead of '-le'
+# (le =label encoded, lb=label binarized, 3d=the 3 continuous features only)
+
+### other datasets : 'shuttle' and 'forestcover'
+
+
+# datasets = ['shuttle', 'forestcover', 'http10-le', 'http10-lb', 'http10-3d',
+#            'smtp-le', 'smtp-lb', 'smtp-3d', 'SF-le', 'SF-lb', 'SF-3d',
+#            'SA-le','SA-lb', 'SA-3d']
+
+datasets = ['http10-le','shuttle', 'forestcover','smtp-le']
```

Comment:
unfortunately that requires manual download of the data. Any chance to use ``fetch_mldata``? Also, for ``forestcover`` aka ``covertype`` we already have functions in ``sklearn.datasets``